tangelo/tangelo/__main__.py

```diff
@@ -194,6 +189,19 @@ def main():
         print tangelo_version
         return 0
 
+    # Make all logging go to stdout instead of stderr.
+    #
+    # The "cherrypy.error" logger contains two handlers by default - a
+    # NullHandler, and a StreamHandler that writes to stderr.  This piece of
+    # code simply removes the StreamHandler, modifies it to write to stdout, and
+    # then replaces it.
+    import logging
+    logger = logging.getLogger("cherrypy.error")
+    handler = logger.handlers[1]
+    logger.removeHandler(handler)
+    handler.stream = sys.stdout
+    logger.addHandler(handler)
+
```

Comment:
I'm not a huge fan of this.  But AFAICT this is how you would need to "redirect" all logging output from ``stderr`` to ``stdout``.

The other option, which has the benefit of being less ugly/potentially fragile, and the drawback of being a sledgehammer-type solution, is to replace this block of code with, simply, ``sys.stderr = sys.stdout`` to do a global, once-and-for-all redirection of everything to ``stdout``.

@jeffbaumes what do you think?